e2e/src/usecases/RestoreAccountOnboarding.js

```diff
@@ -1,90 +1,90 @@
-import { EXAMPLE_NAME, SAMPLE_BACKUP_KEY } from '../utils/consts'
+import { getAddressChunks } from '@celo/utils/lib/address'
+import {
+  EXAMPLE_NAME,
+  SAMPLE_BACKUP_KEY,
+  SAMPLE_BACKUP_KEY_12_WORDS,
+  SAMPLE_WALLET_ADDRESS,
+  SAMPLE_WALLET_ADDRESS_12_WORDS,
+} from '../utils/consts'
 import { launchApp } from '../utils/retries'
-import { enterPinUi, sleep, waitForElementId } from '../utils/utils'
+import { enterPinUi, scrollIntoView, waitForElementId } from '../utils/utils'
 
 export default RestoreAccountOnboarding = () => {
-  beforeAll(async () => {
-    await device.terminateApp()
-    await sleep(5000)
+  beforeEach(async () => {
+    await device.uninstallApp()
+    await device.installApp()
     await launchApp({
-      delete: true,
       permissions: { notifications: 'YES', contacts: 'YES' },
     })
-    await sleep(5000)
-  })
-  // Language is auto selected if it matches one of the available locale
-  // it('Language', async () => {
-  //   await element(by.id('ChooseLanguage/en-US')).tap()
-  // })
-  it('Onboarding Education', async () => {
-    // Onboarding education has 3 steps
-    for (let i = 0; i < 3; i++) {
-      await element(by.id('Education/progressButton')).tap()
-    }
   })
 
-  it('Welcome', async () => {
```

Comment:
i removed these very short test blocks, i'm not sure that they added much value. this let me do the `it.each` more easily on the top level for 12 / 24 word tests, rather than having multiple levels of `describe`s

Reply:
Looks fine, we can also use `describe.each`, or good old loops, to mostly keep the existing structure.
I'll let @MuckT chime in too.

Reply:
i clicked into a few other test files and didn't see any other tests that were broken up so finely (even in the NewAccountOnboarding one) so i thought this file might just be old / outdated from the new patterns. i'd be curious if there is benefit in breaking up tests very finely, i guess the error messages will look different but i think it's still clear where the test failed in either implementation

Reply:
i caught up with @MuckT yesterday and seems like he is fine with removing the existing structure, this is an older pattern that we want to move away from and he mentioned that removing the fine grain tests is more "end to end" here :)